### PR TITLE
10 create userinfo component thierry

### PR DESCRIPTION
**Pull request by peelerzin**
hey,

Here is the pull request for the issue10. I used tailwind config to design the component and created a story for it. No padding is added or whatever because it will be create with the layer above. 

skrrr :)

### CODE REVIEW

File: src/components/UserInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import axios from "axios";
```
Comment (ReallyBoard123) [inline src/components/UserInfo.tsx:1]: delete unused imports

File: src/components/UserInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+type Input = {
+  name: string;
+  image: string;
+};
+
+export function UserInfo({ name, image }: Input) {
+  return (
+    <>
+      <div className="flex flex-row justify-between w-80">
```
Comment (ReallyBoard123) [inline src/components/UserInfo.tsx:13]: why do we need this w-80 here?

File: src/components/UserInfo.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        <div className="flex flex-col justify-between h-10">
+          <div className="flex items-center">
+            <p className="text-secondary text-white-dimmed">Welcome {name}</p>
+            <p>👋</p>
```
Comment (ReallyBoard123) [inline src/components/UserInfo.tsx:17]: this can just be in the same p tag as above, need not make another p tag

**Review by ReallyBoard123 — CHANGES_REQUESTED** [review]
the code looks overall great, just clean the file a bit :)

File: src/components/UserInfo.tsx (hunk 1 above)
Comment (peelerzin) [reply src/components/UserInfo.tsx:1]: deleted.

File: src/components/UserInfo.tsx (hunk 2 above)
Comment (peelerzin) [reply src/components/UserInfo.tsx:13]: deleted only justify-between is needed.

File: src/components/CastCrewButtons.tsx [hunk 4]
```diff
@@ -0,0 +1,14 @@
+export function CastCrewButtons() {
```
Comment (ReallyBoard123) [inline src/components/CastCrewButtons.tsx:1]: This is not part of the #10 please remove it from here, or move it to some other branch

**Review by ReallyBoard123 — CHANGES_REQUESTED** [review]
please do not start other components in the same branch

File: src/components/CastCrewButtons.tsx (hunk 4 above)
Comment (peelerzin) [reply src/components/CastCrewButtons.tsx]: Ok I removed the component and story  CastCrewButtons.

**Review by ReallyBoard123 — APPROVED** [review]

**Pull request merged**